ckan/templates/package/snippets/change_item.html

```diff
@@ -0,0 +1,10 @@
+<strong>On {{ h.render_datetime(activity_diff.activities[1].timestamp, with_hours=True, with_seconds=True) }}, {{ h.linked_user(activity_diff.activities[1].user_id) }}:</strong>
+
+{% set changes = h.compare_pkg_dicts(activity_diff.activities[0].data.package, activity_diff.activities[1].data.package, activity_diff.activities[0].id) %}
+<ul>
+{% for change in changes %}
+  {% snippet "snippets/changes/{}.html".format(
+    change.type), ah=change %}
```

Comment:
I like this idea, except that I wonder if it might make dealing with things like changed tags, extra fields, etc. a bit more difficult? I guess for any case where we need to find a list of changed tags/fields/keys/etc. we could still do that in changes.py and still pass that information to the templates that need it. I'm going to keep it as is for now but I think this would be a good change to make in the future.